packages/datadog-plugin-express/test/index.spec.js

```diff
@@ -214,34 +215,57 @@ describe('Plugin', () => {
-                expect(spans[1]).to.have.property('name', 'express.middleware')
-                expect(spans[1].parent_id.toString()).to.equal(spans[0].span_id.toString())
-                expect(spans[1].meta).to.have.property('component', 'express')
-                expect(spans[2]).to.have.property('resource', 'expressInit')
-                expect(spans[2]).to.have.property('name', 'express.middleware')
-                expect(spans[2].parent_id.toString()).to.equal(spans[0].span_id.toString())
-                expect(spans[2].meta).to.have.property('component', 'express')
-                expect(spans[3]).to.have.property('resource', 'named')
-                expect(spans[3]).to.have.property('name', 'express.middleware')
-                expect(spans[3].parent_id.toString()).to.equal(spans[0].span_id.toString())
-                expect(spans[3].meta).to.have.property('component', 'express')
-                expect(spans[4]).to.have.property('resource', 'router')
-                expect(spans[4]).to.have.property('name', 'express.middleware')
-                expect(spans[4].parent_id.toString()).to.equal(spans[0].span_id.toString())
-                expect(spans[4].meta).to.have.property('component', 'express')
-                expect(spans[5].resource).to.match(/^bound\s.*$/)
-                expect(spans[5]).to.have.property('name', 'express.middleware')
-                expect(spans[5].parent_id.toString()).to.equal(spans[4].span_id.toString())
-                expect(spans[5].meta).to.have.property('component', 'express')
-                expect(spans[6]).to.have.property('resource', '<anonymous>')
-                expect(spans[6]).to.have.property('name', 'express.middleware')
-                expect(spans[6].parent_id.toString()).to.equal(spans[5].span_id.toString())
-                expect(spans[6].meta).to.have.property('component', 'express')
+                expect(spans[index]).to.have.property('resource', 'named')
+                expect(spans[index]).to.have.property('name', 'express.middleware')
+                expect(spans[index].parent_id.toString()).to.equal(rootSpan.span_id.toString())
+                expect(spans[index].meta).to.have.property('component', 'express')
+                index++
+
+                expect(spans[index]).to.have.property('resource', 'router')
+                expect(spans[index]).to.have.property('name', 'express.middleware')
+                expect(spans[index].parent_id.toString()).to.equal(rootSpan.span_id.toString())
+                expect(spans[index].meta).to.have.property('component', 'express')
+                index++
+
+                const handleResource = isExpress4 ? /^bound\s.*$/ : 'handle'
+                if (isExpress4) {
+                  expect(spans[index].resource).to.match(handleResource)
+                } else {
+                  expect(spans[index]).to.have.property('resource', handleResource)
+                }
```

Comment:
Why the double `isExpress4` check?

```suggestion
                if (isExpress4) {
                  expect(spans[index].resource).to.match(/^bound\s.*$/)
                } else {
                  expect(spans[index]).to.have.property('resource', 'handle')
                }
```